XiEditor/EditView.swift

```diff
@@ -389,6 +389,8 @@ class EditView: NSView, NSTextInputClient, TextPlaneDelegate {
         "transpose:": "transpose",
         "yank:": "yank",
         "cancelOperation:": "cancel_operation",
+        "uppercaseWord": "uppercase",
```

Comment:
These need to have trailing colons (confirm that it works when changed, doesn't work for me as-is).